### ISSUE TITLE
[ic-tooltip] label is not announced by a screen reader

### ISSUE THREAD

**Issue by GCHQ-Developer-718** [mi6/ic-ui-kit#3853]
### Summary of the bug 
When an element inside a tooltip is focused, its accessible name is announced by VoiceOver and NVDA but not the tooltip label. As a result, screen reader users don't have access to the additional information provided in the tooltip.

It looks like the element inside the tooltip with `id="id"` has `aria-describedby='ic-tooltip-id"` but no element has an id of ic-tooltip-id. I assume adding this id to the tooltip would fix the issue.

### 🪜 How to reproduce 
Navigate to the tooltip examples on the guidance site using VoiceOver or NVDA and observe that the contents of the tooltip are not announced by either.

### 🧐 Expected behaviour 
The accessible name of the element inside the tooltip should be announced. The label of the tooltip should also be announced as an accessible description.

### 📝 Acceptance Criteria  
The accessible name of the element inside the tooltip is announced. The label of the tooltip is announced as an accessible description.

A new unit test ensures that the aria-described by attribute of the button matches the id of the element providing the description.

The fix is manually tested with VoiceOver and NVDA.

### 🚨 Urgency (low, medium or high)
High
